LICENSE

```diff
@@ -1,6 +1,8 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Haad
+Copyright (c) 2016-2018 Haad
+Copyright (c) 2019 Haja Networks Oy
+
```

Comment:
Unfortunately, these are not correct as this was done under Protocol Labs, so it's:
(c) 2016-2017 Protocol Labs

I believe Haja Networks hasn't made a significant contribution to this to claim on copyright, so I believe it's only that ^. 

cc @RichardLitt

Reply:
Thanks, @haadcode. Changing.